webapp/components/integrations/components/installed_incoming_webhooks.jsx

```diff
@@ -111,6 +111,7 @@ export default class InstalledIncomingWebhooks extends React.Component {
                     creator={this.state.users[incomingWebhook.user_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
@jwilander One thing though correct me if I am wrong. installed_incoming_webhooks already has the dependency of channel here - https://github.com/mattermost/platform/blob/bbf6c81a065d7d2c550b6a629cf31cc113b3713e/webapp/components/integrations/components/installed_incoming_webhooks.jsx#L81 for sorting.

https://github.com/mattermost/platform/blob/bbf6c81a065d7d2c550b6a629cf31cc113b3713e/webapp/components/integrations/components/installed_incoming_webhooks.jsx#L103

And `InstalledIncomingWebhooks` will be a connected component later isn't it? 

Having the dependency of channel store for one each webhook(`InstalledIncomingWebhook`) seemed like an overkill just for one prop as it is available on parent. 

`InstalledIncomingWebhook` as a component can just be a stateless component going forward and will offer a lot of benefits this way.

https://medium.com/missive-app/45-faster-react-functional-components-now-3509a668e69f. This one is a more recent blog entry. 

This one in an old but still makes some good points. https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc. 

I can make the changes you requested if this doesn't make sense let me know what you think.